src/clj/rems/db/test_data.clj

```diff
@@ -693,16 +655,16 @@
     (create-expired-license!)
     (let [dynamic (create-catalogue-item! res1 (:dynamic workflows) form
                                           {"en" "Dynamic workflow" "fi" "Dynaaminen työvuo"})]
-      (create-applications! dynamic (:dynamic workflows) +fake-users+))
+      (create-applications! dynamic +fake-users+))
     (create-catalogue-item! res-with-extra-license (:dynamic workflows) form
                             {"en" "Dynamic workflow with extra license" "fi" "Dynaaminen työvuo ylimääräisellä lisenssillä"})
     (let [thlform (create-thl-demo-form! +fake-users+)
           thl-catid (create-catalogue-item! res1 (:dynamic workflows) thlform {"en" "THL catalogue item" "fi" "THL katalogi-itemi"})]
-      (create-member-applications! thl-catid (:dynamic workflows) (+fake-users+ :applicant1) (+fake-users+ :approver1) [{:userid (+fake-users+ :applicant2)}]))
+      (create-member-applications! thl-catid (+fake-users+ :applicant1) (+fake-users+ :approver1) [{:userid (+fake-users+ :applicant2)}]))
     (let [dynamic-disabled (create-catalogue-item! res1 (:dynamic workflows) form
                                                    {"en" "Dynamic workflow (disabled)"
                                                     "fi" "Dynaaminen työvuo (pois käytöstä)"})]
-      (create-disabled-applications! dynamic-disabled (:dynamic workflows) (+fake-users+ :approver1) (+fake-users+ :approver1))
+      (create-disabled-applications! dynamic-disabled (+fake-users+ :approver1) (+fake-users+ :approver1))
```

Comment:
minor: not introduced by this patch, but shouldn't the second parameter be (+fake-users+ :applicant1)?

Reply:
It probably should, but it breaks many (badly written) tests. Need to refactor those tests first to create their own test data.